AmplifyPlugins/Auth/Tests/AuthHostApp/AuthHostApp/AuthHostApp.entitlements

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
is this change part of the PR or for internal testing?

Reply:
Part of the PR. This is required for integration tests (adding keychain sharing capability to the AuthHostApp)